pkg/controller/minio.go

```diff
@@ -137,11 +140,24 @@ func (c *Controller) checkOperatorCaForTenant(ctx context.Context, tenant *minio
 		return false, err
 	}
 
-	operatorCaCert, err := getOperatorCACert(operatorCaSecret.Data)
+	operatorPublicCert, err := getOperatorCertFromSecret(operatorCaSecret.Data, "public.crt")
 	if err != nil {
+		// If no public.crt is present we error, other certs are optional
 		return false, err
 	}
 
+	certsData["public.crt"] = operatorPublicCert
+
+	operatorTLSCert, err := getOperatorCertFromSecret(operatorCaSecret.Data, "tls.crt")
```

Comment:
can we make the `*.crt` fields, constants ? I see them being used also in `getTransport` so this would help us prevent typos and such.

Reply:
done